test/e2e_env/kubernetes/workload/workload.go

```diff
@@ -57,7 +57,7 @@ func Workload() {
 		Eventually(func(g Gomega) {
 			podName, err = PodNameOfApp(kubernetes.Cluster, appName, namespace)
 			g.Expect(err).ToNot(HaveOccurred())
-		}, "30s", "1s").Should(Succeed())
+		}, "30s").Should(Succeed())
```

Comment:
Sorry for being annoying but 30s is the default :) https://github.com/kumahq/kuma/blob/719105eda57f365c8fefb4581f1a1e142e210e66/pkg/test/ginkgo.go#L42